### PR TITLE
Timur katimullin 1-10 tasks

### PR DESCRIPTION
**Pull request by linkouth**

### CODE REVIEW

File: tasks/2_convert-to-f/convert-to-f.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * convertToF - convert from Celsius to Fahrenheit
+ *
+ * @param {Number} celsius
+ * @return {*}
+ */
+export function convertToF(celsius) {
+  let fahrenheit;
```
Comment (vovcyan) [inline tasks/2_convert-to-f/convert-to-f.js]: Please use const
or
make it in one line: return celsius * 9 / 5 + 32;

File: tasks/7_random-fraction/random-fraction.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
+ * randomFraction - generates random decimal number in range 0 to 1
+ *
+ * @return {Number}
+ */
+export function randomFraction() {
+  return Math.random();
```
Comment (vovcyan) [inline tasks/7_random-fraction/random-fraction.js]: Math.random return some float value, for example 0.16829963774021406
By the task function should return decimal value, for example 0.2

File: training/2_remove_siblings/remove-siblings.spec.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import fs from 'fs';
-import path from 'path';
+//import fs from 'fs';
+// import path from 'path';
```
Comment (vovcyan) [inline training/2_remove_siblings/remove-siblings.spec.js]: Please replace it by:
const fs = require('fs');
const path = require('path');

**Review by vovcyan — CHANGES_REQUESTED** [review]

**Review by vovcyan — DISMISSED** [review]

File: tasks/1_golf-score/golf-score.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ * @return {String}
+ */
+export function golfScore(par, strokes) {
+  if (strokes === 1) {
+    return GOLF_SCORES.HOLE_IN_ONE;
+  }
+  if (strokes <= par - 2) {
+    return GOLF_SCORES.EAGLE;
+  }
+  switch (strokes) {
```
Comment (savchukm) [inline tasks/1_golf-score/golf-score.js:23]: What is the reason of segregating some conditions in different `switch case` construction? At the same time some conditions placed in `if` statements

File: tasks/3_is-boolean/is-boolean.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+/**
+ * convertToF - convert from Celsius to Fahrenheit
+ *
+ * @param {*} bool
+ * @return {Boolean}
+ */
+export function isBoolean(bool) {
+  return typeof(bool) === 'boolean';
```
Comment (savchukm) [inline tasks/3_is-boolean/is-boolean.js]: typeof is an operator - not a function. Parentheses is optional here

File: tasks/1_golf-score/golf-score.js (hunk 4 above)
Comment (linkouth) [reply tasks/1_golf-score/golf-score.js:23]: You are right. The switch statement is redundant there. I fixed it.

File: tasks/3_is-boolean/is-boolean.js (hunk 5 above)
Comment (linkouth) [reply tasks/3_is-boolean/is-boolean.js]: Fixed it

Comment (vovcyan) [reply tasks/3_is-boolean/is-boolean.js]: Btw it's also allowed
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#Syntax

**Review by vovcyan — DISMISSED** [review]

**Review by savchukm — DISMISSED** [review]

File: package-lock.json [hunk 6]
```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```
Comment (alexkond) [inline package-lock.json]: Please revert these changes.

**Review by alexkond — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 6 above)
Comment (linkouth) [reply package-lock.json]: Reverted changes

**Review by vovcyan — DISMISSED** [review]

File: tasks/1_golf-score/golf-score.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+/**
+ * golf-score - returns nickname, depending on
+ * how far above or below par your strokes are.
+ *
+ * @param {Number} par
+ * @param {Number} strokes
+ * @return {String}
+ */
+export function golfScore(par, strokes) {
+  if (strokes === 1) {
```
Comment (alexkond) [inline tasks/1_golf-score/golf-score.js]: I think switch here is more appropriate.

**Review by alexkond — CHANGES_REQUESTED** [review]

File: tasks/1_golf-score/golf-score.js (hunk 7 above)
Comment (linkouth) [reply tasks/1_golf-score/golf-score.js]: There was switch statement. I replaced it by if statements. 
https://github.com/linkouth/ui-internship-lwo2/pull/1#discussion_r322606566

Comment (alexkond) [reply tasks/1_golf-score/golf-score.js]: Well, I've seen this comment, I think @savchukm meant that you shouldn't mix your logic into `switch` and if at the same time. What I mean is that here switch fits nicely instead of a bunch of if blocks.

Comment (linkouth) [reply tasks/1_golf-score/golf-score.js]: Refactored function

**Review by alexkond — APPROVED** [review]

**Review by KalachevDev — APPROVED** [review]
